src/gui/widgets/chatbox.cpp

```diff
@@ -100,9 +100,9 @@ void chatbox::finalize_setup()
 {
 	roomlistbox_ = find_widget<listbox>(this, "room_list", false, true);
 
-	roomlistbox_->set_callback_value_change([this](widget&) {
-		switch_to_window(roomlistbox_->get_selected_row());
-	});
+	// We need to bind a lambda here since switch_to_window is overloaded.
```

Comment:
Please also explain why a lambda alone isn't sufficient. Suggested wording:

    // We need to bind a lambda here since switch_to_window is overloaded.
    // A lambda alone would be more verbose because it'd need to specify all the parameters.